CHANGELOG.md

```diff
@@ -3,3 +3,5 @@
 # v0.49.0 (unreleased)
 - Initial release
 - [RequireFeatureFlagMock](https://github.com/ezcater/ezcater_rubocop/blob/9adbce126ba971b48a9944ef97d2ed6330342632/lib/rubocop/cop/ezcater/rspec_require_feature_flag_mock.rb) Cop
+- [RspecDotNotSelfDot](https://github.com/ezcater/ezcater_rubocop/blob/master/lib/rubocop/cop/ezcater/rspec_dot_not_self_dot.rb) Cop
```

Comment:
I think we should like the the commit hear, not master, since master will change over time... thoughts?

Reply:
Yes, I thought about that when I was adding this. I wouldn't usually bother including links in the CHANGELOG. Also, I was going to squash these commits before merging, so I don't actually know the commit in advance. Unless we want to go back and fill out the CHANGELOG.

What do you think about just removing the links here?

Reply:
We can remove the links... perhaps I was getting ahead of myself when i did that....